packages/camera/camera_avfoundation/ios/Classes/FLTCamMediaSettings.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Foundation;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Media settings configuration parameters.
+ */
+@interface FLTCamMediaSettings : NSObject
+
+/**
+ * @property framesPerSecond optional frame rate of video being recorded.
+ */
+@property(atomic, readonly, strong) NSNumber *framesPerSecond;
```

Comment:
How can all of these properties be non-nullable (per the macro) if the init params are nullable?

Reply:
here and below changed to `@property(atomic, readonly, strong, nullable)`